### PR TITLE
Add Production Machine Learning

### PR DESCRIPTION
**Pull request by axsaucedo**
Added link to Awesome Production Machine Learning https://github.com/EthicalML/awesome-production-machine-learning/

**[Insert URL to the list here]**
https://github.com/EthicalML/awesome-production-machine-learning/

**[Explain what this list is about and why it should be included here]**
It would fit in the machine learning section

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request
All aligned

## Requirements for your Awesome list
All fulfilled

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by axsaucedo** [comment]
Hello @sindresorhus, thanks for the fast comment! I tried to follow the guidelines quite close, could you let me know what the current issue is? Happy to do our best to update it - thak you.

File: readme.md [hunk 1]
```diff
@@ -313,6 +313,7 @@
 	- [H2O](https://github.com/h2oai/awesome-h2o#readme) - Open source distributed machine learning platform written in Java with APIs in R, Python, and Scala.
 	- [Software Engineering for Machine Learning](https://github.com/SE-ML/awesome-seml#readme) - From experiment to production-level machine learning.
 	- [AI in Finance](https://github.com/georgezouq/awesome-ai-in-finance#readme) - Solving problems in finance with machine learning.
+	- [Production ML](https://github.com/EthicalML/awesome-production-machine-learning/#readme) - Frameworks to deploy, monitor & scale machine learning
```
Comment (keller-mark) [inline readme.md:316]:
```suggestion
	- [Production ML](https://github.com/EthicalML/awesome-production-machine-learning/#readme) - Frameworks to deploy, monitor & scale machine learning.
```

You may need to add a period to the end of your description here. Also the description may need to be more like

```
Deploy, monitor & scale machine learning with frameworks.
```

**Review by keller-mark — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (axsaucedo) [reply readme.md]: Thanks @keller-mark, I've added a period

**Comment by keller-mark** [comment]
Also you may want to read the requirements closer. There is an “easter egg”-type of thing that you may have missed near the end.

**Comment by axsaucedo** [comment]
Thanks @keller-mark,just had a look at the guidelines again but can't find anything that's not in place, what's currently missing?

**Comment by makma** [comment]
Hi @axsaucedo, I'm not sure but I believe the guys above wanted to point out that your entry should be added **at the bottom** of the appropriate category.

**Comment by axsaucedo** [comment]
Thanks for the heads up @makma - are you sure? Just had a look at it, and it is at the bottom of the appropriate category (although in this case it's a sub-category). Or you mean it's currently in the wrong category? If so I am happy to move it for sure

**Comment by axsaucedo** [comment]
I'm not clear what's the guideline that is not being met - @sindresorhus would you be able to provide more details?

**Comment by makma** [comment]
> Thanks for the heads up @makma - are you sure? Just had a look at it, and it is at the bottom of the appropriate category (although in this case it's a sub-category). Or you mean it's currently in the wrong category? If so I am happy to move it for sure

You're right, it's my bad, I've read the indentation incorrectly. I'm sorry about that.

**Comment by makma** [comment]
@axsaucedo -  Another try - maybe the unicorn thingie?

**Comment by axsaucedo** [comment]
Thank you @makma - what is the unicorn?

**Comment by axsaucedo** [comment]
Reviewed https://github.com/sindresorhus/awesome/pull/1865

**Comment by axsaucedo** [comment]
Reviewed https://github.com/sindresorhus/awesome/pull/1857

**Comment by axsaucedo** [comment]
unicorn

**Comment by axsaucedo** [comment]
@sindresorhus @keller-mark anything else I should make sure to cover?

**Comment by driesvints** [comment]
Cool awesome list. One improvement I would suggest is unifying the type of badge used across the list to indicate the Github stars.

Screenshot 2020-09-17 at 10 41 52

**Comment by axsaucedo** [comment]
@sindresorhus @keller-mark anything else I should make sure to cover?

File: readme.md [hunk 2]
```diff
@@ -313,6 +313,7 @@
 	- [H2O](https://github.com/h2oai/awesome-h2o#readme) - Open source distributed machine learning platform written in Java with APIs in R, Python, and Scala.
 	- [Software Engineering for Machine Learning](https://github.com/SE-ML/awesome-seml#readme) - From experiment to production-level machine learning.
 	- [AI in Finance](https://github.com/georgezouq/awesome-ai-in-finance#readme) - Solving problems in finance with machine learning.
+	- [Production ML](https://github.com/EthicalML/awesome-production-machine-learning/#readme) - Frameworks to deploy, monitor & scale machine learning.
```
Comment (sindresorhus) [inline readme.md:316]:
```suggestion
	- [Production ML](https://github.com/EthicalML/awesome-production-machine-learning#readme) - Frameworks to deploy, monitor & scale machine learning.
```

**Comment by axsaucedo** [comment]
Thanks @sindresorhus - I've added the suggested changes, let me know if further additions are required

**Comment by sindresorhus** [comment]
Please run https://github.com/sindresorhus/awesome-lint and fix what you can.

**Comment by sindresorhus** [comment]
The list heading should be title-cased.

**Comment by sindresorhus** [comment]
> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

**Comment by sindresorhus** [comment]
Don't repeat the project name at the start of the description:

> pygal is a dynamic SVG charting library written in python

**Comment by sindresorhus** [comment]
> Consistent and correct naming. For example, Node.js, not NodeJS or node.js.

**Comment by sindresorhus** [comment]
> The description starts with an uppercase character and ends with a period.

**Comment by sindresorhus** [comment]
Bump

**Comment by Ly0n** [comment]
The linter still shows me a lot of formatting errors that could easily be avoided. The displayed GitHub stars make the list hard to read. When a project is listed as Awesome, you don't have to show how "Awesome" it is. However, this is my personal opinion and is okay with the contribution guide so far.
The projects listed are actually very good and active.

**Pull request closed without merging**